adapters/handlers/grpc/v1/filters.go

```diff
@@ -194,9 +194,9 @@ func extractDataTypeProperty(getClass func(string) *models.Class, operator filte
 		}
 
 		classOfProp := on[len(on)-2]
-		class := getClass(classOfProp)
-		if class == nil {
-			return dataType, fmt.Errorf("could not find class %s in schema", classOfProp)
+		class, err := getClass(classOfProp)
+		if err != nil {
+			return schema.DataTypeString, err
```

Comment:
Should this not be `return dataType, err` to align with how the other error branches are defined in this func?